src/Xamarin.Android.Build.Tasks/Tests/Xamarin.ProjectTools/Android/XamarinAndroidApplicationProject.cs

```diff
@@ -175,7 +175,9 @@ private bool PublishAot {
 		/// </summary>
 		public void SetPublishAot (bool value, string androidNdkPath)
 		{
-			IsRelease = value;
+			// Only toggle IsRelease=true when value is true
+			if (value)
+				IsRelease = true;
```

Comment:
If I want to say  unit test a NativeAOT app in debug mode is this going to stop that from working?

Reply:
I would maybe still test `Release` configuration and set `$(DebugType)` and `$(DebugSymbols)`.

Eventually, I think `$(PublishAot)=true` will still use Mono (or CoreCLR), so our managed debuggers will work.